modules/server_new/src/main/scala/observe/server/syntax.scala

```diff
@@ -64,11 +64,10 @@ extension [F[_]](q: ExecutionQueue)
     )
   def clear: ExecutionQueue                                             = q.copy(queue = List.empty)
 
-implicit final class ToHandle[F[_]: Applicative, A](f: EngineState[F] => (EngineState[F], A)) {
-  import Handle.toHandle
-
+import Handle.{toHandle => toHandleT}
```

Comment:
Maybe this should g inside the extension?

Reply:
I tried but got an error. `'def' expected, but 'import' found`